_includes/download-buttons-design.html

```diff
@@ -1,12 +1,9 @@
-{% comment %}
-These `uswds-assets-*.zip` files are generated in 18F/web-design-standards-assets
-repository. See: https://github.com/uswds/uswds-for-designers
-{% endcomment %}
 <div class="link__group-download">
   <a class="link-download"
-    href="https://github.com/uswds/uswds-for-designers/releases/download/v2.6.0/uswds-for-designers-v2.6.0.zip"
-    onclick="ga('send', 'event', 'Downloaded design files - Sketch', 'Clicked download design files button inside site');">Download the USWDS {{ site.uswds_version }} Design Kit (Sketch) [ZIP; 12.4 MB]</a>
-  <a class="link-download"
-    href="https://github.com/uswds/uswds-for-designers/releases/download/v2.6.0/uswds-for-designers-v2.6.0.zip"
-    onclick="ga('send', 'event', 'Downloaded design files - Adobe XD', 'Clicked download design files button inside site');">Download the USWDS {{ site.uswds_version }} Design Kit (Adobe XD) [ZIP; 12.4 MB]</a>
+    href="https://github.com/uswds/uswds-for-designers/releases/download/v3.0.0/uswds-for-designers-v3.0.0.zip"
+    onclick="ga('send', 'event', 'Downloaded design files', 'Clicked download design files button inside site');">
```

Comment:
This will work once we publish `uswds-for-designers`. I've collected everything into a single link. The previous iteration had two links that led to the same file only to collect an analytics event. This type of information isn't worth the UX hit